app/main.py

```diff
@@ -1,8 +1,44 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
```

Comment:
Use descriptive and correct variable names. The variable names `comfort_class` and `clean_mark` are somewhat descriptive, but they could potentially be more specific to better represent their purpose within the context of a `Car`.